db/migrate/20140623100942_create_visits.rb

```diff
@@ -1,7 +1,7 @@
 class CreateVisits < ActiveRecord::Migration
   def change
     create_table :visits do |t|
-      t.uuid :visitor_id
+      t.integer :visitor_id
```

Comment:
Why?

Reply:
activeUUID is incompatible with Rails 5, so when it was removed the the uuid type no longer existed. Initially I replaced it with the integer type because that's ahoy's default and because of #1229. I've since replaced it with binary(16), since that's what it actually is, according to the activeUUID code